docs/eventing/samples/kafka/channel/README.md

```diff
@@ -151,3 +151,72 @@ Now you can see the events in the log of the `ksvc` using the command:
 ```
 kubectl logs --selector='serving.knative.dev/service=broker-kafka-display' -c user-container
 ```
+
+## Authentication against an Apache Kafka broker
+
+The KafkaSource supports TLS and SASL authentication methods.
+
+### TLS authentication
+
+For enabling TLS authentication, please have the below files
+
+* CA Certificate
+* Client Certificate and Key
+
+`KafkaChannel` expects these files to be in pem format, if it is in other format like jks, please convert to pem.
+
+
+1. Create the certificate files as secrets in the namespace of your choice, like:
+```
+$ kubectl create secret --namespace knative-eventing generic my-tls-secret \
+  --from-file=ca.crt=caroot.pem \
+  --from-file=user.crt=certificate.pem \
+  --from-file=user.key=key.pem
+ ```
```

Comment:
Won't let me suggest properly, but I'd make some tweaks here:


1. Create the certificate files as secrets in your chosen namespace:

```
$ kubectl create secret --namespace <namespace> generic <secret-name> \
  --from-file=ca.crt=caroot.pem \
  --from-file=user.crt=certificate.pem \
  --from-file=user.key=key.pem
 ```

Reply:
@matzew doesn't look like this one was updated, or the one below, e.g. "like..." and "Now all you need is to reference..." are very fluffy, that's why I suggested making these more minimal and fluff-free

Reply:
I think it was updated:
https://github.com/knative/docs/pull/3067/files#diff-4259c7eeed66d201ebf025c842c68c0ad1a5c75121bde0869b80dc21ed7a14e8R170-R180

OK.. I did not see initially the suggestion about "Create the certificate files as secrets in your chosen namespace:" - will change